src/routes/Books.ts

```diff
@@ -0,0 +1,47 @@
+import Router from '@koa/router';
+import { Book, PostBook } from '../models/Books'
+import db from '../db'
+const bookRouter = new Router();
+const BOOK_ENDPOINT = "/books"
+const ID = "/:id"
+
+bookRouter
+  .post(BOOK_ENDPOINT, async (ctx) => {
+    const data = <PostBook>ctx.request.body
+    const book = new Book();
+    book.name = data.name
+    book.isbn = data.isbn
+    book.author = data.author
+    book.releaseDate = data.releaseDate
+    await db.manager.save(book);
+
```

Comment:
Do not forget to return some response or you'll get 404 for this route in the response. You can return created book e.g.:
```ts
const book = await db.manager.save(book);
res.json(book);
```